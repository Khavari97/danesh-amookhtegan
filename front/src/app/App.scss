@import 'assets/fonts/fonts';

* {
  margin: 0;
  font-family: "Shabnam", serif;
}

a {
  text-decoration: none;
}

//Slider css
.alice-carousel__dots {
  .alice-carousel__dots-item:not(.__custom).__active {
    background-color: #046B00;
  }
}


