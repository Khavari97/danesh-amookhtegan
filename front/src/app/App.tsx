import React from "react";
import "./App.scss";
import AppRouter from "./navigation/AppRouter";
import {SnackbarProvider} from 'notistack';
import {SnackbarUtils} from "./components/material-ui/Snackbar/SnackbarUtils";
import ThemeProvider from "./assets/styles/material/material";

export interface AppProps {
}

export interface AppState {
}

class App extends React.Component<AppProps, AppState> {
    constructor(props: AppProps) {
        super(props);
        this.state = {};
    }

    componentDidMount = () => {
    }

    render() {
        return (
            <SnackbarProvider maxSnack={3} anchorOrigin={{horizontal: 'right', vertical: 'bottom'}}>
                <SnackbarUtils/>
                <ThemeProvider>
                    <AppRouter/>
                </ThemeProvider>
            </SnackbarProvider>
        );
    }
}

export default App;
