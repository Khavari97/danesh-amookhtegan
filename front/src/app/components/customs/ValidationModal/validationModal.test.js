import React from "react"
import Enzyme, { shallow,configure,mount } from "enzyme";
import renderer from "react-test-renderer";
import Adapter from 'enzyme-adapter-react-16';
import App from './ValidationModal';

configure({ adapter: new Adapter() })

const setup = (props = {}) => {
  return shallow(<App {...props} />);
};

const findByTestAttr = (wrapper, val) => {
    return wrapper.find(`[data-test="${val}"]`);
};

describe('render ValidationModal without error',()=>{
    let wrapper = setup();
    it('should have value',()=>{
        wrapper.setProps({
            title: 'string',
            description: 'string',
            open: 'boolean',
          })
        expect(findByTestAttr(wrapper,'title').text()).toBe("string");
        expect(findByTestAttr(wrapper,'description').text()).toBe("string");
        expect(findByTestAttr(wrapper,'button').text()).toBe("بله");
        expect(findByTestAttr(wrapper,'button').prop('color')).toBe("primary");
    })

    it('snapshot',()=>{
        const shot = <App />
        const snapshot = renderer.create(shot).toJSON();
        expect(snapshot).toMatchSnapshot();
    })
})