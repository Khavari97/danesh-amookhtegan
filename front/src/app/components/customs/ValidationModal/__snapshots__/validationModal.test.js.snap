// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ValidationModal without error snapshot 1`] = `null`;
